src/main/java/edu/greenblitz/robotName/Robot.java

```diff
@@ -123,4 +124,18 @@ private void initializeLogger() {
         }
         Logger.start();
     }
+    public static RobotType getRobotType (){
+        RobotType robotType = RobotConstants.ROBOT_TYPE;
+        if(isReal()){
```

Comment:
What is “isReal” ?

Reply:
Where is the code located?

Reply:
in the parent class of the Robot.java class